src/Analyzers/CSharp/Tests/UseConditionalExpression/UseConditionalExpressionForAssignmentTests.cs

```diff
@@ -1953,5 +1949,68 @@ private bool AreSimilarCore(string node1, string node2)
                 }
                 """, LanguageVersion.CSharp9, equivalenceKey: nameof(AnalyzersResources.Simplify_check));
         }
+
+        [Theory, WorkItem("https://github.com/dotnet/roslyn/issues/67649")]
+        [InlineData("int", "int")]
+        [InlineData("string", "string")]
+        [InlineData("string", "object")]
+        [InlineData("object", "string")]
```

Comment:
can you do one like `[InlineData("int", "long")]`?  I want to make sure that things work well for 'implicit', but not 'reference', conversions.